cime_config/buildlib.csm_share

```diff
@@ -6,12 +6,12 @@ use File::Path;
 # Purpose: Build csm_share library
 #===============================================================================
 
-if ($#ARGV == -1) {
-    die " ERROR buildlib_csm_share: must specify a caseroot input argument";
+if ($#ARGV < 2) {
```

Comment:
This check and the associated 'die' message look wrong: it looks like 3 arguments are actually expected